### PR TITLE
controller-tools: update admins and maintainer group

### PR DESCRIPTION
**Pull request by vincepri**

### CODE REVIEW

**Review by alvaroaleman — APPROVED** [review]

**Review by chaodaiG — APPROVED** [review]

**Review by camilamacedo86 — APPROVED** [review]
/lgtm 


Hi @deads2k, 

How are you? I hope that you are fine.
Could you please give an hand with this one?

File: config/kubernetes-sigs/sig-api-machinery/teams.yaml
Comment (deads2k) [inline config/kubernetes-sigs/sig-api-machinery/teams.yaml:77]: @camilamacedo86 still looks actively involved in this project.  Please restore her.

Comment (vincepri) [reply config/kubernetes-sigs/sig-api-machinery/teams.yaml:77]: While active, this isn't in line with the https://github.com/kubernetes-sigs/controller-tools/blob/master/OWNERS_ALIASES and its history; this PR just sync the actual group permissions with what's reflected in the repository.

Comment (deads2k) [reply config/kubernetes-sigs/sig-api-machinery/teams.yaml:77]: It looks like a similar paperwork maneuver was used to update that file recently too: https://github.com/kubernetes-sigs/controller-tools/pull/799

Who are the people who actually maintain controller-tools?  A dev-stats screenshot for controller-tools perhaps?

Comment (vincepri) [reply config/kubernetes-sigs/sig-api-machinery/teams.yaml:77]: Going by the metrics [here](https://k8s.devstats.cncf.io/d/13/developer-activity-counts-by-repository-group?orgId=1&var-period_name=Last%203%20years&var-metric=prs&var-repogroup_name=All&var-repo_name=kubernetes-sigs/controller-tools&var-country_name=All)

PR Created: 
CleanShot 2023-05-18 at 11 46 08@2x

PR Reviewed:
CleanShot 2023-05-18 at 11 46 52@2x

PR Approved:
CleanShot 2023-05-18 at 11 47 16@2x

Comment (camilamacedo86) [reply config/kubernetes-sigs/sig-api-machinery/teams.yaml:77]: Hi @deads2k, 

I am not in the Owner alias, and I am totally fine about removing myself from here. 

My primary focus is on Kubebuilder. Unfortunately, I do not have the bandwidth to contribute as much as I’d like to all projects. These permissions were initially granted by Solly and others due to the significant dependence of Kubebuilder on these projects, when Solly step aside. I never have been in the Owner alias. Those were added to add intended to enable us to manage emergency releases or fast-track specific issues as required swiftly.

@vincepri @joelanford  has been doing the releases, etc. So, it is totally fine. 

Btw, thank you for all consideration

File: config/kubernetes-sigs/sig-api-machinery/teams.yaml
Comment (vincepri) [inline config/kubernetes-sigs/sig-api-machinery/teams.yaml:64]: These folks have moved to emeritus a while back: https://github.com/kubernetes-sigs/controller-runtime/blob/260b822e9daa53b2ddec007ec48359e38a3ef797/OWNERS_ALIASES#L39-L41

File: config/kubernetes-sigs/sig-api-machinery/teams.yaml
Comment (vincepri) [inline config/kubernetes-sigs/sig-api-machinery/teams.yaml:65]: Not sure about @seans3, it doesn't show up in any commits in https://github.com/kubernetes-sigs/controller-tools/commits/master/OWNERS_ALIASES? This might precede my time in this repo

File: config/kubernetes-sigs/sig-api-machinery/teams.yaml
Comment (vincepri) [inline config/kubernetes-sigs/sig-api-machinery/teams.yaml:67]: This group was removed given that effectively doesn't provide any value, and it doesn't seem referenced anywhere

File: config/kubernetes-sigs/sig-api-machinery/teams.yaml
Comment (camilamacedo86) [reply config/kubernetes-sigs/sig-api-machinery/teams.yaml:77]: Hi @deads2k, 

I am not in the Owner alias, and I am totally fine OK with removing myself from here. 

My primary focus is on Kubebuilder. Unfortunately, I do not have the bandwidth to contribute as much as I’d like to all projects.Btw, thank you for all your consideration
 
@vincepri and @joelanford have been taking care of the project so I think that is right.

File: config/kubernetes-sigs/sig-api-machinery/teams.yaml
Comment (vincepri) [inline config/kubernetes-sigs/sig-api-machinery/teams.yaml:80]: @mengqiy 👋 seems like Solly added you [in this commit](https://github.com/kubernetes-sigs/controller-tools/commit/0662bea6af62ef389c27a543381bdf5175235ab6) which was made alongside this other [commit in controller-runtime](https://github.com/kubernetes-sigs/controller-runtime/commit/71e9a1626803390e880d668142f062b613572956). Since then @mengqiy was moved to emeritus after inactivity in this [controller-runtime commit](https://github.com/kubernetes-sigs/controller-runtime/commit/344e96e1accc8bed25d09bfd8bd935bb44943081)

Comment (vincepri) [reply config/kubernetes-sigs/sig-api-machinery/teams.yaml:80]: The above seems also true for @droot

File: config/kubernetes-sigs/sig-api-machinery/teams.yaml
Comment (deads2k) [reply config/kubernetes-sigs/sig-api-machinery/teams.yaml:77]: > Hi @deads2k,
> 
> I am not in the Owner alias, and I am totally fine OK with removing myself from here.

Ok.  According to Vince's devstats above, you are the top code reviewer for the repo, but if it's not something you have the bandwidth that's understandable.

Comment (vincepri) [reply config/kubernetes-sigs/sig-api-machinery/teams.yaml:77]: Absolutely, @camilamacedo86 contributions in controller-tools have been more than invaluable for the project; the purpose of updating this list is mostly to keep the write permissions to the repository itself scoped to the smallest subset of folks that need it, mostly to push tags or do releases in GitHub.

We can always add a `controller-tools-reviewers` group like other projects have (incl. controller-runtime), which we haven't done yet as nobody stepped up and didn't come up before, how does that sound?

File: config/kubernetes-sigs/sig-api-machinery/teams.yaml
Comment (fedebongio) [inline config/kubernetes-sigs/sig-api-machinery/teams.yaml:82]: Sean (seans3) is planning to get involved again shortly (he actually stepped down from SIG CLI). Please keep seans3 for now. 
I had the same concern about removing Camila but saw her response too.

Comment (vincepri) [reply config/kubernetes-sigs/sig-api-machinery/teams.yaml:82]: Done ptal

Comment (vincepri) [reply config/kubernetes-sigs/sig-api-machinery/teams.yaml:82]: Also opened a PR in the repository to add @seans3 as maintainer, and add the old maintainers to emeritus, which I missed the first time we cleaned things up https://github.com/kubernetes-sigs/controller-tools/pull/817

Comment (vincepri) [reply config/kubernetes-sigs/sig-api-machinery/teams.yaml:82]: /hold

@alvaroaleman @sbueringer @joelanford

ptal to confirm these changes

**Review by alvaroaleman — APPROVED** [review]

File: config/kubernetes-sigs/sig-api-machinery/teams.yaml
Comment (sbueringer) [reply config/kubernetes-sigs/sig-api-machinery/teams.yaml:82]: @fedebongio As far as I can tell Sean doesn't have any contributions to controller-tools as of now (according to devstats).

While we are definitely happy over any new contributors, I would have assumed that usually folks are going through the contribution ladder (https://github.com/kubernetes/community/blob/master/community-membership.md) before gaining maintainer privilages (including approve & write access to the repository).

Apologies if I'm missing something.

Comment (vincepri) [reply config/kubernetes-sigs/sig-api-machinery/teams.yaml:82]: Opened https://github.com/kubernetes/org/pull/4239 as a follow-up once Sean starts contributing to the project.

@fedebongio ptal

**Comment by mrbobbytables** [comment]
/hold cancel
/approve
(approver has lgtmed above)

**Pull request merged**
